.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
We should remove this directory or add it to .gitignore

Reply:
I miss this file, sorry 😝